### PR TITLE
Sign In to Phantom

### PR DESCRIPTION
**Pull request by kelliaUmuhire**
#### What does this PR do?
* Adds Sign In functionality
#### Description of Task to be completed?
* User should be able to login using email and password
#### How should this be manually tested?
* Clone the repo
* Run npm install to install modules
* Run npm run dev to start the development server and the database
* Go to the endpoint `"/api/auth/signin"`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker/Trello stories?
* [Link](https://trello.com/c/2yZQOVnv)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Phantom (backend)
 
+<<<<<<< HEAD
+Elevate the public transport experience
```
Comment (dmutoni) [inline README.md:4]: Can you please remove these <<<<

File: README.md [hunk 2]
```diff
@@ -54,9 +58,15 @@ Execute the command `npm run test` to run all tests
 
 ## Contribute
 
+<<<<<<< HEAD
+After cloning and running the project successfully, follow these steps:
```
Comment (dmutoni) [inline README.md:62]: And here

File: src/controllers/userController.ts [hunk 3]
```diff
@@ -60,4 +64,49 @@ const findUserByEmail = async (email: string) => {
...
+
+export const signIn = (req: Request, res: Response) => {
+  UserModel.findOne({
+    where: {
+      email: req.body.email,
+    },
+  })
+    .then((user) => {
+      if (!user) {
+        return res.status(404).send({ message: "User Not found." });
```
Comment (dmutoni) [inline src/controllers/userController.ts]: I would prefer it if you return the response using API_RESPONSE so as to make it uniform

File: src/controllers/userController.ts [hunk 4]
```diff
@@ -60,4 +64,49 @@ const findUserByEmail = async (email: string) => {
...
+      }
+
+      let passwordIsValid = bcrypt.compareSync(
+        req.body.password,
+        user.password
+      );
+
+      if (!passwordIsValid) {
+        return res.status(401).send({
```
Comment (dmutoni) [inline src/controllers/userController.ts]: and here I mean the response

File: test/user.spec.ts [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+// let should = chai.should();
+
+// chai.use(chaiHttp);
+
+// let t_user = {
+//   email: "tester@phantom.com",
+//   password: "w6vav7",
+// };
+
```
Comment (dmutoni) [inline test/user.spec.ts:21]: Why are these lines commented?

**Review by Phinah — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kelliaUmuhire) [reply README.md:4]: On it!

File: src/controllers/userController.ts (hunk 3 above)
Comment (kelliaUmuhire) [reply src/controllers/userController.ts]: Sure, that would be better.

**Pull request merged**